### ISSUE TITLE
Cannot recognize multi task

### ISSUE THREAD

**Issue by webconsult** [firstandthird/load-grunt-config#149]
Hi,

So far I've been enjoying using your plugin a lot, but I struggle setting up a multitask with the plugin `grunt-text-replace`.

I'm trying to set up a workflow for svgs and I've created a small test repo for the purpose here:

https://github.com/webconsult/svgtest

Whenever I run grunt I get an error stating that `Warning: Task "replace:illustrator" not found. Use --force to continue.`

In the process of debugging this problem I rewrote everything to not use `load-grunt-config` and if you look in my gruntfile.js you will see this configuration as commented out lines. If you comment those lines in and comment out the lines that were not commented out, you will see that the configuration works fine without `load-grunt-config`.

When I took the config and copied it out from my gruntfile.js to grunt/replace.js, the only change I made was that I removed the outer wrapper `replace: { ... }`. In my experience that has been necessary whenever I've written a config in a separate file like that. I tried leaving it in, but if I run `grunt --config-debug` I see that there are two layers of `replace` nested within each other, so it seems it should be removed. Here is the output of grunt --config-debug, by the way:

https://gist.github.com/webconsult/ce40df51db8ed6fc5056

Any suggestions what I'm doing wrong here?

**Comment by SolomoN-ua** [comment]
Hi @webconsult,

after quick check I can't find `grunt-text-replace` in your `package.json`, according to [load-grunt-tasks](https://github.com/sindresorhus/load-grunt-tasks) documentation (we are using this plugin inside of `load-grunt-configs`):
>This module will read the dependencies/devDependencies/peerDependencies/optionalDependencies in your package.json and load grunt tasks that match the provided patterns.

So if it is not in package.json, this plugin will be not loaded. (I guess it is a root cause of your issue)
Please try to fix your package.json, and let me know if you still have problems. :)

**Comment by webconsult** [comment]
You're right. I experienced the problem on another project, though, and created this one as a sandbox to debug the problem. How I've managed to not install `grunt-text-replace` I don't know, but that was not the case on the actual project were I was first experiencing this error.

I did manage to solve the error in the "real" project, though. I had called the config file ´grunt/text-replace.js`. After renaming it to ´grunt/replace.js´ things were working again.
